nixos/modules/services/networking/headscale.nix

```diff
@@ -462,6 +462,22 @@ in {
       isSystemUser = true;
     };
 
+    services.nginx = {
+      enable = true;
```

Comment:
This needs to be optional - I don't use nginx, and the web server I do use sits on a separate VM to headscale.